sdk/servicebus/azure-servicebus/azure/servicebus/_common/constants.py

```diff
@@ -115,6 +115,11 @@ class SessionFilter(Enum):
     NextAvailable = 0
 
 
+class SubQueue(Enum):
+    DeadLetter = 1
+    TransferDeadLetter = 2
```

Comment:
I have some thoughts on the naming, but we could discuss in the next sprint.

like:
```python
class SubQueueType(Enum):
    DeadLetterQueue = 1
    TransferDeadLetterQueue = 2
```

Reply:
See the poll that Ramya ran on this exact question :)  (I voted for SubQueueType, for the record :P )

Reply:
lol, we're the minority.
well.. I can live with it. not a big issue